standard.md

```diff
@@ -0,0 +1,136 @@
+- **tool**: `str`. Name of the tool used to prepare the data for the diagnostic.
+- **version**: `str` Version of the tool used to prepare the data for the diagnostic.
+- **run_dir**: `str`. Path to the directory to use as current path for the diagnostic execution. *Must* be writable by the diagnostic.
+- **data_dir**: `str`. Path to the directory to store the  diagnostic's generated data on. *Must* be writable by the diagnostic.
+- **plot_dir**: `str`. Path to the directory to store the  diagnostic's generated figures on. *Must* be writable by the diagnostic.
+
+It is *suggested* to provide different directory names for `run_dir`, `data_dir` and `plot_dir`, as it makes the interaction with the results
+easier. However, using the same directory for all purposes is accepted.
+
+### Reserved options
+
+The following options are not required, however if present they must follow the definitions listed below:
+
+- **log_level**: `str`, default value is `info`. Sets the granularity of the diagnostic logs and console output.
+If used, the logger must include `error` `warning`, `info` and `debug` levels .
+- **auxiliary_data_dir** : `str`. Path to the auxiliary directory to store further input data files needed by the diagnostic, such as shapefiles.
+- **max_proc_number** : `integer`, default value is **1**. Maximum number of processors used to run the diagnostic.
+
+
+## The data definition file
+
+The data definition file *must* consist of a YAML file containing a mapping of mappings. The top-level mapping will have as keys the path to the data files. And as values, the mapping containing the metadata of each file.
+
+
+### Required options
+
+- **alias**: `str`. Unique name for the dataset. Each (`alias`, `variable`) pair *must* be unique.
+- **filename**: `str`. Absolute path to the data file.
+- **variable**: `str`. Variable name to be used in the diagnostic. It is not necessary to follow CF conventions naming for variables.
+
+### Reserved options
+
+#### Related to the dataset
+
+- **project**: `str`. Name of the project to which a dataset belongs. It can be a real project, (e.g. `CMIP5`, `CMIP6`) or a tool-defined grouping of datasets
+(e.g. `Observations` or `Reanalysis`).
+- **activity**: `str`. CMIP activity, or analog, of a dataset.
+- **institute**: `str`. Name of the institute producing a dataset.
+- **dataset**: `str`. Name of the dataset. For example: the CMIP model identifier, the name of an observational dataset, etc.
+- **experiment**: `str`. Label to identify the experiment a dataset belongs to. (e.g `historical`, `piControl`).
+- **ensemble**: `str`. Ensemble identifier of a dataset.
+
+#### Related to the variable
+
+- **table**: `str`. Variable's original MIP table.
+- **original_frequency**: `str`. Variable's original frequency.
+- **frequency**: `str`. Variable's frequency when entering the diagnostic.
+- **modeling_realm**: `str`. Realms a variable belongs to.
+- **grid**: `str`. Variable's original grid.
```

Comment:
I think this means the CMIP6 grid label, not the actual grid, right? It would be good to write this down more clearly.

Reply:
done